scripts/update/0.5-0.6/cluster.sh

```diff
@@ -0,0 +1,143 @@
+#!/bin/bash
+
+# This script:
+# 1) Appends the label `lokomotive.alpha.kinvolk.io/bgp-enabled=true` in the env file for the nodes
+#    running MetalLB.
+# 2) Update the image tag of Kubelet.
+# 3) Update etcd if it is a controller node.
+
+set -euo pipefail
+
+mode=$1
+
+function run_on_host() {
+  nsenter -a -t 1 /bin/sh -c "${1}"
+}
+
+updated_kubelet_config=false
+packet_environment=false
+
+function is_packet_environment() {
+  if grep -i packet /run/metadata/flatcar > /dev/null; then
+    packet_environment=true
+  fi
+}
+
+function update_kubelet_env_file() {
+  readonly kubelet_env="/etc/kubernetes/kubelet.env"
+  readonly kubelet_version="v1.19.4"
+
+  if grep "${kubelet_version}" "${kubelet_env}" >/dev/null; then
```

Comment:
TODO: Don't just rely on the kubelet version, also check for the label. It is quiet possible that a user might have an updated kubelet version if they have openebs installed and have followed: https://github.com/kinvolk/lokomotive/blob/master/docs/how-to-guides/troubleshoot-openebs.md#step-32-update-kubelet-version